WooCommerce/Classes/ViewRelated/Coupons/CouponListViewController.swift

```diff
@@ -270,6 +271,24 @@ private extension CouponListViewController {
     }
 }
 
+
+// MARK: - Placeholder cells
+//
+extension CouponListViewController {
+    /// Renders the Placeholder Coupons
+    ///
+    func displayPlaceholderCoupons() {
+        displayGhostContent()
+    }
+
+    /// Removes the Placeholder Coupons
+    ///
+    func removePlaceholderCoupons() {
+        removeGhostContent()
+    }
+}
+
+
```

Comment:
nit: replace 2 new lines with 1 above and below

Reply:
That looks like a clean revert of the removal [here](https://github.com/woocommerce/woocommerce-ios/commit/287a44e97a51cfa3d10b1f8f53f804154928d53d#diff-dd1109897bde5c30d5266dc80305497fc0fba3a4fe7b156cc52a80ff242551c1).

Let's stick with what's there, the changes produced by reverting the merge commit, rather than introducing new changes on top of those.